src/api/app/models/gitea_payload/pull_request.rb

```diff
@@ -6,7 +6,6 @@ def payload
       commit_sha: webhook_payload.dig(:pull_request, :head, :sha),
       pr_number: webhook_payload[:number],
       source_branch: webhook_payload.dig(:pull_request, :head, :ref),
-      target_branch: webhook_payload.dig(:pull_request, :base, :ref),
```

Comment:
removing only this makes little sense...

Reply:
Yes, I reverted this